src/cascadia/TerminalSettingsEditor/Microsoft.Terminal.Settings.Editor.vcxproj

```diff
@@ -283,7 +289,6 @@
     <ProjectReference Include="$(OpenConsoleDir)src\cascadia\TerminalConnection\TerminalConnection.vcxproj">
       <Private>false</Private>
     </ProjectReference>
-

```

Comment:
nit: mind adding all these newlines back in?

Reply:
I think VS removed them but sure, I added them back in.